portfolio/src/main/java/com/google/sps/servlets/UserSubmittedLocationsServlet.java

```diff
@@ -70,21 +70,24 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Seems like nothing is hitting this endpoint with a POST yet?

Reply:
the action on the place button form has a POST action within it, so it is directly creating a post request when you click submit